app/src/main/java/com/github/sdpteam15/polyevents/view/activity/admin/EventManagementActivity.kt

```diff
@@ -1,13 +1,356 @@
 package com.github.sdpteam15.polyevents.view.activity.admin
 
+import android.annotation.SuppressLint
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
+import android.widget.*
 import androidx.appcompat.app.AppCompatActivity
 import com.github.sdpteam15.polyevents.R
+import com.github.sdpteam15.polyevents.helper.HelperFunctions
+import com.github.sdpteam15.polyevents.model.database.remote.Database.currentDatabase
+import com.github.sdpteam15.polyevents.model.entity.Event
+import com.github.sdpteam15.polyevents.model.entity.Zone
+import com.github.sdpteam15.polyevents.model.observable.Observable
+import com.github.sdpteam15.polyevents.model.observable.ObservableList
+import java.time.LocalDateTime
 
 class EventManagementActivity : AppCompatActivity() {
+
+    companion object {
+        const val MIN_PART_NB = 1
+        const val emptyPartNb = "0"
+        const val typeStart = "start"
+        const val typeEnd = "END"
+        private const val defaultPartNb = "10"
```

Comment:
maybe it's a good idea to use the same convention for the constente

Reply:
Totally right, it's changed in the last version :)